.vscode/settings.default.json

```diff
@@ -9,7 +9,6 @@
 	"typescript.format.enable": false,
 	"typescript.tsdk": "node_modules/typescript/lib",
 	"workspace-default-settings.runOnActivation": true,
-	"prettier.prettierPath": "node_modules/prettier",
```

Comment:
See [here](https://github.com/n8n-io/n8n-hosted-backend/pull/508#discussion_r1270740407).

Reply:
I downgraded prettier. not sure if 2.8.x still needs this setting.

Reply:
Removal still needed at 2.8.x

<img width="1440" alt="image" src="https://github.com/n8n-io/n8n/assets/44588767/58b9f79a-5dd9-4703-b406-c643a227280f">

Reply:
every time prettier is upgraded or downgraded, you might need to restart vscode.